cpp/velox/utils/ConfigExtractor.cc

```diff
@@ -34,6 +34,12 @@ const bool kVeloxFileHandleCacheEnabledDefault = false;
 // Log granularity of AWS C++ SDK
 const std::string kVeloxAwsSdkLogLevel = "spark.gluten.velox.awsSdkLogLevel";
 const std::string kVeloxAwsSdkLogLevelDefault = "FATAL";
+// Retry mode for AWS s3
+const std::string kVeloxS3RetryMode = "spark.gluten.velox.fs.s3a.retry.mode";
+const std::string kVeloxS3RetryModeDefault = "legacy";
+// Connection timeout for AWS s3
+const std::string kVeloxS3ConnectTimeout = "spark.gluten.velox.fs.s3a.connect.timeout";
+const std::string kVeloxS3ConnectTimeoutDefault = "1s";
```

Comment:
can we align the default config value with the hadoop config. https://hadoop.apache.org/docs/r3.0.0/hadoop-project-dist/hadoop-common/core-default.xml , such as the default value for `fs.s3a.connection.timeout` is 200000ms

Reply:
`200s` seems quite a large number. @FelixYBW what do you think?

Reply:
Let's use hadoop's default one